argilla-frontend/components/features/home/dataset-list/DatasetList.vue

```diff
@@ -48,10 +48,10 @@ export default {
     return {
       querySearch: "",
       sortedOrder: "desc",
-      sortedByField: "updatedAt",
+      sortedByField: "lastActivityAt",
       sortOptions: [
         { value: "name", label: this.$t("home.name") },
-        { value: "updatedAt", label: this.$t("home.updatedAt") },
+        { value: "lastActivityAt", label: this.$t("home.updatedAt") },
```

Comment:
I was thinking about this change, should we still having the same label?

Reply:
Yes, we have mentioned keeping “updated” to express “last_activity_at”, we can maintain the label